permissions/plugin-github-checks.yml

```
@@ -0,0 +1,9 @@
+---
+name: "github-checks"
+github: "jenkinsci/github-checks-plugin"
+paths:
+- "org/jenkins-ci/plugins/github-checks-plugin"
```

```suggestion
- "io/jenkins/plugins/github-checks"
```

This info is taken from the pom.xml (groupdId/artifactId): 
https://github.com/jenkinsci/github-checks-plugin/blob/master/pom.xml#L13
https://github.com/jenkinsci/github-checks-plugin/blob/master/pom.xml#L14

---

Sorry, just followed other templates without a second thought and check...